src/sage/rings/polynomial/ore_polynomial_ring.py

```diff
@@ -613,13 +644,7 @@ def _repr_(self) -> str:
             sage: T
             Ore Polynomial Ring in d over Univariate Polynomial Ring in t over Rational Field twisted by d/dt
         """
-        s = "Ore Polynomial Ring in %s over %s twisted by " % (self.variable_name(), self.base_ring())
-        if self._derivation is None:
-            s += self._morphism._repr_short()
-        else:
-            if self._morphism is not None:
-                s += "%s and " % self._morphism._repr_short()
-            s += self._derivation._repr_()
+        s = "Ore Polynomial Ring in %s over %s %s" % (self.variable_name(), self.base_ring(), self._repr_twist())
         if self.is_sparse():
```

Comment:
These days, f-strings are popular. It is used like
```
s = f"Ore Polynomial Ring in {self.variable_name()} over {self.base_ring()} {self._repr_twist()}"
```
Just a mild suggestion.

Reply:
I know. But I'm used to using the syntax with `%`.
If you insist, I can switch.

Reply:
I don't insist. 

OK.